CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to React Native Sample APP
+
+Please follow this guidelines to start contributing to this project.
+
+# Creating a pull request
+
+1. Create a new branch from **main** branch following the name convention.
+2. Use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) convention for commit messages.
+3. Create a PR and include any useful context of the change in the description. Include any relevant screenshots.
+4. Wait for the PR to be reviewed and approved.
+
+# Branch naming convention
+
+The branch name should include the category, the Jira ticket number and a short description using "-" as separator
+
+Use the following format: {category}/{Jira tickket number}-{short description}
+
+| Category  | Meaning                                             |
+|-----------|-----------------------------------------------------|
+| hotfix    | for quickly fixing critical issues                  |
```

Comment:
I am actually using 'commitizen' plugin on VSCode which is kind of similar to this but the naming is different. For example hotfix is 'fix'. There are also other namings such as 'refactor', 'doc', 'style'... 
I am up to follow the standard, do you guys use a plugin for it or just write prefix on every commit ?

Reply:
made some small changes